### DISCUSSION TITLE
How to style the icon ">" after each link

### DISCUSSION THREAD

**Discussion by Hael616** [saadeghi/daisyui#2204] [Q&A] [answered]
Hi there, please is there a way to style the ">" icon after each link, I'm using an all white background and I can barely see it

**Comment by saadeghi** [comment, marked as answer]
It should be visible:  
<img width="885" alt="2" src="https://github.com/saadeghi/daisyui/assets/7342023/86979935-5e41-4209-a07f-244611384187">

If it's not visible it may be another issue 🤔

However you can change the style with a CSS like this  
https://play.tailwindcss.com/iW4uUT6Ekv?file=css
```css
.breadcrumbs > ul > li + *:before {
  opacity: 1;
}
```
Or with a utility class like this  
https://play.tailwindcss.com/dMMlEVUXOG
```
[&_ul_li+*:before]:opacity-100
```

Let me know if you have any questions
